amplify.yml

```diff
@@ -3,8 +3,10 @@ frontend:
   phases:
     preBuild:
       commands:
-        - nvm install 18.13
-        - nvm use 18.13
+        - NODE_VERSION=$(cat .tool-versions | grep '^nodejs' | sed 's/nodejs //')
+        - echo "Using Node.js version $NODE_VERSION"
+        - nvm install "$NODE_VERSION"
+        - node -v
```

Comment:
This is great! It nwill allow the non-Cru orgs to not have to edit this file if they have a different version